src/linkgraph/linkgraphjob.cpp

```diff
@@ -106,15 +106,15 @@ LinkGraphJob::~LinkGraphJob()
 		/* The station can have been deleted. Remove all flows originating from it then. */
 		Station *st = Station::GetIfValid(from.base.station);
 		if (st == nullptr) {
-			this->EraseFlows(node_id);
+			this->EraseFlows(from.base.station);
```

Comment:
Doesn't changing from `node_id` to `from.base.station` make this a Fix rather than a Codefix? Or did it happen to have the same value?